### PR TITLE
Some fixes.

### PR DESCRIPTION
**Pull request by squeakyspacebar**
Some fixes.  Let me know if anything looks janky.

### CODE REVIEW

**Comment by paultag** [comment]
@loandy I see three different changes - there's some style stuff, some stdin stuff and the fix for passing the ValueError up and out - can you split it into three PRs?

File: moxie/cores/ssh.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def exit(stdin, stdout, stderr, args=None):
 def readl(stdin, stdout, echo=True):
     buf = ""
     while not stdin.at_eof():
-        bytes_ = (yield from stdin.read())
+        bytes_ = (yield from stdin.read(1))
```
Comment (paultag) [inline moxie/cores/ssh.py:79]: This is going to be super thrashy, it's going to result in a lot of coroutine jitter - I'd at least read a larger hunk, IIRC, read() will shot-read, that's just the max size of the read

Comment (squeakyspacebar) [inline moxie/cores/ssh.py:79]: The interface would hang as soon as I connected, so this was the fix I put in.  I'll take it down and mess with it a little before resubmitting.

**Comment by squeakyspacebar** [comment]
I can go ahead and split it.

File: moxie/cores/ssh.py (hunk 1 above)
Comment (paultag) [inline moxie/cores/ssh.py:79]: Try `.read(4096)`

Comment (paultag) [inline moxie/cores/ssh.py:79]: I'm like 90% sure I can out-type the `.read(1)` approach there :)

**Pull request closed without merging**